javascriptlib/src/mill/javascriptlib/PublishModule.scala

```diff
@@ -116,9 +91,25 @@ trait PublishModule extends TypeScriptModule {
       os.symlink(T.dest / ".npmrc", npmInstall().path / ".npmrc")
   }
 
-  override def compile: T[(PathRef, PathRef)] = Task {
+  override def compile: T[PathRef] = Task {
     pubSymLink()
-    super.compile()
+    createNodeModulesSymlink()
+
+    val out = super.compile().path
+
+    os.walk(
+      out,
+      skip = p =>
+        p.last == "node_modules" ||
+        p.last == "package-lock.json" ||
+        p.last == pubBundledOut()
+        p.last == declarationDir()
```

Comment:
What are we doing here? Skipping these based on their `.last` looks wrong

Reply:
mmm, there is no need to do this anyhow `tsc` would just overwrite it if the folder exists